assignment4/test/lib/views/search_screen/SearchedItem.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:contacts_service/contacts_service.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:get/get_core/src/get_main.dart';
+
+import '../../controller/getx_controller.dart';
+import '../detail_screen/detail_screen.dart';
+
+class SearchedItem extends StatefulWidget {
```

Comment:
Why are you using Stateful widget here? Try to avoid Stateful widgets when you're already using GetX for state management

Reply:
changed to stateless widget . initially i was not knowing too much about Getx thats why i used Stateful.Now its resolved